UI/aoc-app/src/Components/SignUpForm.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
```

Comment:
this file seems to just be a duplicate of `CreateProfile.jsx`. This should render that view when a cleaner creates an account through OAuth

Reply:
Yeah, I acidentally staged this file. I'll remove and go from there.